### PR TITLE
Bezugsrahmenwechsel LV03 to LV95

### PR DESCRIPTION
**Pull request by nadine-piveteau**
Changes in OL3 related to the transition of reference system from LV03 to LV95

### CODE REVIEW

**Comment by nadine-piveteau** [comment]
When I resolve these conflicts, errors occur after having built the app (`make -f Makefile-ga build-ga`). @oterral : Any suggestions? Thx in advance!

File: examples/ga-custom.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ var map = new ga.Map({
     // 650, 500, 250, 100, 50, 20, 10, 5, 2.5, 2, 1, 0.5, 0.25, 0.1
     resolution: 10,
     // Define a coordinate CH1903 for the center of the view
```
Comment (oterral) [inline examples/ga-custom.js]: LV95 instead of CH1903

Comment (nadine-piveteau) [reply examples/ga-custom.js]: Yes, I just realized it :) thx!

**Comment by nadine-piveteau** [comment]
Example [GeoAdmin feature recenter example](http://localhost:9020/build/examples/ga-feature.html?q=ga-feature) does not work yet because of this [line](https://github.com/geoadmin/ol3/pull/147/commits/2c2062ce767805aea5b2a7b921a9003d8eaad91c#diff-50c0f385028b9c34a0f113ec24458490L27) in `examples/ga-features.js`. It appears that function `recenterFeature` zooms to a feature in the dataset `ch.swisstopo.fixpunkte-agnes` (featureId = 'PAYE') which is most probably still in LV03. Via a REST request this information is fetched [here](https://github.com/geoadmin/ol3/blob/ltpin_lv95/src/ga/map.js#L172).

@oterral : Do you agree that nothing needs to be changed in OL in this regard?

**Comment by oterral** [comment]
Yes nothing in OL.

You need to add the parameter '&sr=2056'  [here](https://github.com/geoadmin/ol3/blob/ltpin_lv95/src/ga/map.js#L145), [here](https://github.com/geoadmin/ol3/blob/ltpin_lv95/src/ga/map.js#L174) and [here](https://github.com/geoadmin/ol3/blob/ltpin_lv95/src/ga/map.js#L205)

To test, you must hardcode the service url [here](https://github.com/geoadmin/ol3/blob/ltpin_lv95/src/ga/map.js#L111) to use th LV 95branch  : 
```
this.serviceUrl = "https://mf-chsdi3.dev.bgdi.ch/gal_lv95/"
```

**Comment by nadine-piveteau** [comment]
Great, it works! Thx! I'll push it right away

**Comment by nadine-piveteau** [comment]
Just to be sure: `ga-tms.js` doesn't need any modification as it uses pixel indices and not a spatial reference system. @oterral : You agree?

**Comment by oterral** [comment]
I agree

File: examples/ga-feature.js [hunk 2]
```diff
@@ -13,8 +13,8 @@ var map = new ga.Map({
     // List of resolution of the WMTS layers:
     // 650, 500, 250, 100, 50, 20, 10, 5, 2.5, 2, 1, 0.5, 0.25, 0.1
     resolution: 10,
-    // Define a coordinate CH1903 for the center of the view
-    center: [561666.5, 185569.5]
+    // Define a coordinate CH1903+ for the center of the view
```
Comment (oterral) [inline examples/ga-feature.js]: LV95

File: examples/ga-no-tooltip.js [hunk 3]
```diff
@@ -26,7 +26,7 @@ var vector = new ol.layer.Vector({
   source: new ol.source.Vector({
     url: 'data/kml/bln-style.kml',
     format: new ol.format.KML({
-      projection: 'EPSG:21781'
+      projection: 'EPSG:2056'
```
Comment (oterral) [inline examples/ga-no-tooltip.js]: you can remove it. According to the doc it's not used.

File: examples/ga-vector-layer.js [hunk 4]
```diff
@@ -13,7 +13,7 @@ var vector = new ol.layer.Vector({
   source: new ol.source.Vector({
     url: 'data/cities.geojson',
     format: new ol.format.GeoJSON(),
-    projection: 'EPSG:21781'
+    projection: 'EPSG:2056'
```
Comment (oterral) [inline examples/ga-vector-layer.js]: Here too `projection` is not used .

File: src/ga/layer/layer.js [hunk 5]
```diff
@@ -279,8 +279,8 @@ ga.source.wmts = function(layer, options) {
         '" target="new">' +
         options['attribution'] + '</a>')
     ],
-    url: ('http://wmts{5-9}.geo.admin.ch/1.0.0/{Layer}/default/{Time}/21781/' +
-        '{TileMatrix}/{TileRow}/{TileCol}.').replace('http:',location.protocol) + extension,
+    url: ('http://tod{0-4}.prod.bgdi.ch/1.0.0/{Layer}/default/{Time}/2056/' +
```
Comment (oterral) [inline src/ga/layer/layer.js]: @gjn what urls do we plan to use for the go live of lv95 ? wmts{5-9} as before ?

**Comment by oterral** [comment]
Thx @nadine-piveteau for the nice work!!!

**Pull request merged**